tests/admin_views/tests.py

```diff
@@ -6275,6 +6276,152 @@ def test_hidden_fields_small_window(self):
+            """,
+        )
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(LIVING_COUNTRY_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            <option value="1">Argentina</option>
+            <option value="2" selected="">Spain</option>
+            """,
+        )
+        # Spain won't appear because `favorite_country_to_vacation` field
+        # has `limit_choices_to`
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(FAVORITE_COUNTRY_TO_VACATION_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            """,
+        )
+
+        # Edit SECOND Country created from LIVING select
+        favorite_select = Select(
+            self.selenium.find_element(By.ID, LIVING_COUNTRY_SELECT_ID)
+        )
+        favorite_select.select_by_visible_text("Spain")
+        self.selenium.find_element(By.ID, f"change_{LIVING_COUNTRY_SELECT_ID}").click()
+        self.wait_for_and_switch_to_popup()
+        favorite_name_input = self.selenium.find_element(By.ID, "id_name")
+        favorite_name_input.clear()
+        favorite_name_input.send_keys("Italy")
+        self.selenium.find_element(By.CSS_SELECTOR, '[type="submit"]').click()
+        self.selenium.switch_to.window(self.selenium.window_handles[0])
+
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(BORN_COUNTRY_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            <option value="1" selected="">Argentina</option>
+            <option value="2">Italy</option>
+            """,
+        )
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(LIVING_COUNTRY_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            <option value="1">Argentina</option>
+            <option value="2" selected="">Italy</option>
+            """,
+        )
+        # `favorite_country_to_vacation` field is still without options
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(FAVORITE_COUNTRY_TO_VACATION_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            """,
+        )
+
+        # Add a new Asian country
+        self.selenium.find_element(
+            By.ID, f"add_{FAVORITE_COUNTRY_TO_VACATION_SELECT_ID}"
+        ).click()
+        self.wait_for_and_switch_to_popup()
+        favorite_name_input = self.selenium.find_element(By.ID, "id_name")
+        favorite_name_input.send_keys("Qatar")
+        continent_select = Select(
+            self.selenium.find_element(By.ID, CONTINENT_SELECT_ID)
+        )
+        continent_select.select_by_visible_text("Asia")
+        self.selenium.find_element(By.CSS_SELECTOR, '[type="submit"]').click()
+        self.selenium.switch_to.window(self.selenium.window_handles[0])
+
+        # Submit the new Traveler
```

Comment:
Please add trailing periods to your comments. Comments are sentences.

Reply:
I added trailing periods in other places too!

Let me know If forgot any!